src/movies/api/GetTopRated.js

```diff
@@ -0,0 +1,10 @@
+//Top Rated
+export default async function TopRatedMovie(topRatedMovie) {
+  try {
+    let response = await fetch(topRatedMovie);
+    response = await response.json();
+    return response;
+  } catch (error) {
+    console.log("Something went wrong:   ", error);
+  }
+}
```

Comment:
Nome della funzione non troppo chiaro essendo una funziona che torna dei dati io ti consiglio nel rinominare la funziona in getTopRatedMovie (le maiuscole iniziali lasciamole ai componenti)

Reply:
Ho pensato di rinominare la funzione "getMovie" perchè non vale solo per i top rated ma posso usarla per ottenere ogni dato che mi occorre (ad esempio attualmente la sto usando anche per ottenere i dati dei film nel nuovo branch), andrebbe bene?